src/sagemaker/tensorflow/deploying_tensorflow_serving.rst

```diff
@@ -269,6 +269,194 @@ More information on how to create ``export_outputs`` can be found in `specifying
 refer to TensorFlow's `Save and Restore <https://www.tensorflow.org/guide/saved_model>`_ documentation for other ways to control the
 inference-time behavior of your SavedModels.
 
+Providing Python scripts for pre/pos-processing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can add your customized Python code to process your input and output data. You can add your customized Python code to process your input and output data:
+
+.. code::
+
+    from sagemaker.tensorflow.serving import Model
+
+    model = Model(entry_point='inference.py',
+                  model_data='s3://mybucket/model.tar.gz',
+                  role='MySageMakerRole')
+
+How to implement the pre/post-processing handler function
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Your entry point file should implement either a pair of ``input_handler``
+   and ``output_handler`` functions or a single ``handler`` function.
+   Note that if ``handler`` function is implemented, ``input_handler``
+   and ``output_handler`` will be ignored.
+
+To implement pre/post-processing handler(s), you will need to make use
+of the ``Context`` object created by Python service. The ``Context`` is
+a ``namedtuple`` with following attributes:
+
+-  ``model_name (string)``: the name of the model you will to use for
```

Comment:
"the name of the model to use for inference"

simple present tense when possible

Reply:
Thanks. I reviewed the text and tried to apply simple present.